### DISCUSSION TITLE
GITHUB API - Trouble getting comments from a pull request

### DISCUSSION THREAD

**Discussion by benna100** [community/community#24791] [Apps, API and Webhooks]
<p>I cannot get the comment from babakbandpey on this pull request: <a href="https://github.com/masumesani/hyf-homework/pull/12" rel="nofollow noopener">https://github.com/masumesani/hyf-homework/pull/12</a></p>
<p><a href="https://api.github.com/repos/masumesani/hyf-homework/pulls/12/comments" rel="nofollow noopener">https://api.github.com/repos/masumesani/hyf-homework/pulls/12/comments</a></p>
<p><a href="https://api.github.com/repos/masumesani/hyf-homework/issues/12/comments" rel="nofollow noopener">https://api.github.com/repos/masumesani/hyf-homework/issues/12/comments</a></p>
<p>Can anyone help?</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>HI <a class="mention" href="/benna100">@benna100</a>,</p>
<p>Thank you for being here! <a href="https://developer.github.com/v3/pulls/comments/">Pull Request Review Comments</a> are comments on a portion of the unified diff whereas <a href="https://developer.github.com/v3/issues/comments/">Pull Request Comments</a> are comments which do not reference a portion of the unified diff.</p>
<p>Based on your repo, the following command should return all comments on pull request 12.</p>
<p><a href="https://api.github.com/repos/masumesani/hyf-homework/pulls/12/reviews">https://api.github.com/repos/masumesani/hyf-homework/pulls/12/reviews</a></p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/6/640ec96920563629068df6fb447ede67e16b0c02.png" data-download-href="/uploads/short-url/eh9qvr2ZO0k4fk1HGO18w4hfgno.png?dl=1" title="Screen Shot 2020-02-25 at 4.54.14 PM.png"><img src="https://user-images.githubusercontent.com/20666190/181086884-1eb60cc8-abb6-4190-b55c-c8d1159d034f.png" alt="Screen Shot 2020-02-25 at 4.54.14 PM.png" data-base62-sha1="eh9qvr2ZO0k4fk1HGO18w4hfgno" width="660" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/6/640ec96920563629068df6fb447ede67e16b0c02_2_660x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/6/640ec96920563629068df6fb447ede67e16b0c02_2_990x750.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/6/640ec96920563629068df6fb447ede67e16b0c02.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/6/640ec96920563629068df6fb447ede67e16b0c02_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2020-02-25 at 4.54.14 PM.png</span><span class="informations">1093×827 64.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by benna100** [comment]
<p>Awesome, just what i was looking for. Thank you for your help :smiley:</p>
